### ISSUE TITLE
Inclusion in Minetest

### ISSUE THREAD

**Issue by Wuzzy2** [FaceDeer/update_translations#19]
I have just started a discussion on Minetest about including this script as an official utility:

https://github.com/minetest/minetest/issues/10852

I think it's good enough now. :-)

**Comment by ronoaldo** [comment]
Very nice! Is this available in 5.4? I missed it somehow 😬

**Comment by Wuzzy2** [comment]
No, it has not been included. So far, this is only an issue, not even a PR.

**Comment by ronoaldo** [comment]
Sorry - my confusion. I saw some "Merged" related to the issue linked and mixed things up! Thanks!
